.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to Stockfish
+
+Welcome to the Stockfish project! We're excited that you're interested in contributing. This document outlines the guidelines and steps to follow when making contributions to Stockfish.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Setting up Stockfish](#setting-up-stockfish)
+  - [Building Stockfish](#building-stockfish)
+- [Making Contributions](#making-contributions)
+  - [Reporting Issues](#reporting-issues)
+  - [Submitting Pull Requests](#submitting-pull-requests)
+- [Code Style](#code-style)
+- [Community and Communication](#community-and-communication)
+- [License](#license)
+
+## Getting Started
+
+### Setting up Stockfish
+
+1. Fork this repository.
+2. Clone your forked repository: `git clone https://github.com/your-username/Stockfish.git`
+3. Add the Stockfish repository as a remote: `git remote add upstream https://github.com/official-stockfish/Stockfish.git`
+
+### Building Stockfish
+
+In case you dont have a C++ Compiler installed you can follow the instructions from our wiki [linux](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#linux)/[windows](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#windows)/[macos](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#macos) to install one.
```

Comment:
```suggestion
In case you don't have a C++ compiler installed you can follow the instructions from our wiki [Linux](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#linux)/[Windows](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#windows)/[macOS](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#macos) to install one.
```